src/Makefile

```diff
@@ -929,6 +929,9 @@ net: netvariables
 		fi; \
 	fi; \
 
+format: .depend
+	awk '{for(i=1;i<=NF;i++) print $$i}' .depend | grep -v '[\\:]$$' | xargs realpath | sort | uniq | grep -v incbin | xargs clang-format -i -style=file
```

Comment:
Different clang-format versions will end up with slightly different formatted code (using `make format` with clang-format 17 RC 4 results in 11 modified lines). So we might want to prefer a fixed version (e.g. `clang-format-16`) and fall back to `clang-format` if the fixed version doesn't exist. Not sure if it's worth the effort though.